src/__test__/Detail.test.js

```diff
@@ -1,21 +1,21 @@
-import React from "react";
-import { render, screen } from "@testing-library/react";
-import { Provider } from "react-redux";
-import thunk from "redux-thunk";
-import configureStore from "redux-mock-store";
-import { BrowserRouter } from "react-router-dom";
-import Detail from "../pages/Detail";
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+import configureStore from 'redux-mock-store';
+import { BrowserRouter } from 'react-router-dom';
+import Detail from '../pages/Detail';
 
 const mockStore = configureStore([thunk]);
 
-describe("Detail Component", () => {
-  it("renders game details properly", () => {
+describe('Detail Component', () => {
+  it('renders game details properly', () => {
```

Comment:
- [ ] Hi @naolkidanu, please check your tests as @OyePriscilla stated. I see about 5 tests failing with only 1 passing. This will save you a lot of time and submission opportunities.

![image](https://github.com/naolkidanu/game-web-app/assets/99715640/bc98c38f-3595-46ff-a41e-bf6d9513c8a3)